### PR TITLE
Add ability to provide defaults for filters.

### PR DESCRIPTION
**Pull request by nickescobedo**
This pull request allows someone to specify a default value by adding `->default(value)` to a filter.

### CODE REVIEW

**Comment by nickescobedo** [comment]
Anything I can do to help get this in?

**Comment by pulkitjalan** [comment]
This would be a great feature to have.

File: src/Filter.php [hunk 1]
```diff
@@ -91,6 +98,23 @@ public function getIgnored(): array
...
+
+        return $this;
+    }
+
+    public function getDefault()
+    {
+        return $this->default;
+    }
+
+    public function defaultSet(): bool
```
Comment (dominikb) [inline src/Filter.php]: Maybe you could add a verb to the method name, to better fit in with other "check" methods: `isDefaultSet()`

Comment (nickescobedo) [reply src/Filter.php]: Done.

**Comment by nickescobedo** [comment]
@dominikb any chance the changed you asked for could be looked at? 😃 
Thanks for your work!

**Comment by nickescobedo** [comment]
@AlexVanderbist or @freekmurze any chance this PR could be reviewed?

Thanks! 😄

**Comment by AlexVanderbist** [comment]
Hi! Sorry to keep you guys waiting - I'm gonna merge this into the v2 branch that'll be released later this week/beginning of next week. Thanks for the PR!

**Comment by AlexVanderbist** [comment]
Had to merge this one manually into the `v2` branch but it's in there. Thanks again!

**Pull request closed without merging**